build.gradle

```diff
@@ -23,6 +23,7 @@ buildscript {
         }
 
         common_utils_version = System.getProperty("common_utils.version", opensearch_build)
+        kotlin_version = System.getProperty("kotlin.version", "1.8.21")
```

Comment:
We can remove this as alerting tool moved to skills repo

Reply:
Done.